src/zenml/integrations/kubernetes/orchestrators/manifest_utils.py

```diff
@@ -196,6 +196,14 @@ def add_pod_settings(
     for container in pod_spec.containers:
         assert isinstance(container, k8s_client.V1Container)
         container._resources = settings.resources
+        if settings.volume_mounts:
+            container.volume_mounts = settings.volume_mounts
```

Comment:
For both this and the `volumes` beflow, we need to check whether we've already set that before (e.g. a local directory when running locally)